scripts/base/frameworks/input/main.zeek

```diff
@@ -24,10 +24,10 @@ export {
 		STREAM = 2
 	};
 
-	## The default input reader used. Defaults to `READER_ASCII`.
+	## The default input reader used. Defaults to ``READER_ASCII``.
```

Comment:
:zeek:see: ?

and below for `default_mode` ?

Reply:
It's always bothered me that Zeek puts the enum values into the module's namespace instead of one below that, but changing that would be a massive change. For example `Input::MANUAL` instead of `Input::Mode::MANUAL`.